keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -427,6 +439,15 @@ parametersRef:
   name: acme-gpu-init
 ```
 
+#### Integration with cluster autoscaling
+
+As a cloud provider, I want to support GPUs as part of a hosted Kubernetes
+environment, including cluster autoscaling. I ensure that the kernel is
+configured as required by the hardware and that the container runtime supports
+CDI. I review the Go code provided by the vendor for simulating cluster scaling
```

Comment:
AFAIK vast majority of users use the default Cluster Autoscaler image. Certainly not every cloud provider supports their own image - I don't have any hard data here, but I believe most don't.